content/embeds/docker-memory-limitation.md

```diff
@@ -0,0 +1,3 @@
+If you cannot activate the database because of a memory limitation,
+make sure that Docker has enough memory allocated in the Advanced section
+of Docker Settings.
```

Comment:
Please use the appropriate convention to call out "Advanced" and "Docket Settings". This is a place where it'd be useful to link to the official Docker documentation.

Reply:
Unfortunately it's different per OS. Do you think we should have links for each OS?
Currently it's written for Windows which itself is a problem.
I would lean toward making it more generic and that customers should go consult the Docker docs if they need to. I guess we can link to https://docs.docker.com/.